tests/cache/sampleStore.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/cache/sampleStore.js
+ */
+'use strict'; // eslint-disable-line strict
+const sampleStore = require('../../cache/sampleStore');
+const samstoinit = require('../../cache/sampleStoreInit');
+const redisClient = require('../../cache/redisCache').client.sampleStore;
+const expect = require('chai').expect;
+const tu = require('../testUtils');
+const u = require('./utils');
+const Aspect = tu.db.Aspect;
+const Subject = tu.db.Subject;
+const Sample = tu.db.Sample;
+
+describe('sampleStore (feature off):', () => {
```

Comment:
This test fails if redis feature is enabled in environment. Maybe we should add before after like this:
  before((done) => {
    tu.toggleOverride(sampleStore.constants.featureName, false);
    done();
  });

  after((done) => {
    tu.toggleOverride(sampleStore.constants.featureName, true);
    done();
  });